pkg/scyllaclient/client_scylla.go

```diff
@@ -1125,6 +1125,28 @@ func (c *Client) ControlTabletLoadBalancing(ctx context.Context, enabled bool) e
 	return err
 }
 
+// RaftReadBarier triggers read barrier for the given Raft group to wait for previously committed commands in this group to be applied locally.
+// For example, can be used on group 0 (default, when group is not provided) to wait for the node to obtain latest schema changes.
+func (c *Client) RaftReadBarier(ctx context.Context, groupID string, timeout int64) error {
+	params := operations.RaftReadBarrierPostParams{
+		Context: ctx,
+	}
```

Comment:
`RaftReadBarier` -> `RaftReadBarrier` (small typo)

I think that I might have confused you earlier.
`RaftReadBarrier` has a scope of a single node - it ensures the node that gets this API call applies previously committed raft commands - but it does not ensure it for other nodes.
It means that the node that gets `RaftReadBarrier` API call and the node that we query for `DESC SCHEMA WITH INTERNALS` needs to be the same node (we use a single host gocql session in order to ensure that both `DESC SCHEMA` and cql raft read barrier were routed to the same node).

You can use `forceHost` in order to ensure that.

Reply:
Yes, make sense. For some reason I thought that client is already initialized to use particular host - I will fix that :)